doc/source/api/metadata.rst

```diff
@@ -31,6 +31,82 @@ provider, users should not make semantic assumptions about that data.
 Subsets of the data in a dataset can be selected for analysis using
 other metadata or attributes.
 
+.. _metadata_date_time:
+
+Date and Time Format Specifications
+-----------------------------------
+
+Date and time formats are specified as ISO8601 compatible strings, both for
+time points as well as for intervals and durations.
+An optional required granularity may be specified as part of the respective
+attributes' documentations.
+
+Time points
+===========
+
+The specification of a time point is given through the concatenation of
+
+* a date in YYYY-MM-DD
+* the designator "T" indicating a following time description
+* the time of day in HH:MM:SS.SSS form, where "SSS" represents a decimal
+  fraction of a second
+* a time zone offset in relation to UTC
+
+**Examples**
+
+* year (YYYY)
+    2015
+
+* date (e.g. date of birth) in YYYY-MM-DD
+    2015-02-10
+
+* time stamp in milliseconds in YYYY-MM-DDTHH:MM:SS.SSS
+    2015-02-10T00:03:42.123Z
+
+**Implementations**
+
+* created (G2P, metadata)
```

Comment:
Since ```created```, ```updated``` are now universal, the specific branch pointers to G2P, metadata should be removed.